.changeset/petite-pears-make.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Is it normal to have .changeset/forty-games-enter.md and .changeset/petite-pears-make.md for the same scope and patch? Can’t we merge them?

Reply:
While it is rare, multiple changesets in a given PR are supported. They are usually added if additional changes were made outside the scope of the issue but related or touched code.